roles/maps/defaults/main.yml

```diff
@@ -70,83 +82,46 @@ maps_dot_black_vector_tiles:
   # "high res" full osm, including 3d buildings. 78GiB as of Oct 2025
   # (TODO does this include colors and topography? Or is it used along with naturalearth6 above in most styles?)
   # maps_vector_quality = "osm-full"
-  osm-full:
-    name: openstreetmap-openmaptiles-full.pmtiles
-    filename: openstreetmap-openmaptiles-full.pmtiles # NOTE - saves as different filename than the source URL to make room for the symlink
-    src_url: "{{ iiab_map_host_url }}/openstreetmap-openmaptiles.pmtiles"
-    sha256sum: 95b90ef180b15f6496d01f7437a9ed058d2a1b61ac5dba32f43790010acba77c
+  osm-full: "openstreetmap-openmaptiles.{{ maps_data_date }}.full.pmtiles"
 
   # "medium res" osm, up to zoom level 9 (original file has 14). 1.9GiB as of Oct 2025
   # (TODO does this include colors and topography? Or is it used along with naturalearth6 above in most styles?)
   # maps_vector_quality = "osm-z9"
-  osm-z9:
-    name: openstreetmap-openmaptiles-zoom_0-09.pmtiles
-    filename: openstreetmap-openmaptiles-zoom_0-09.pmtiles
-    src_url: "{{ iiab_map_host_url }}/openstreetmap-openmaptiles-zoom_0-09.pmtiles"
-    sha256sum: 32fc190bd9009c54b8fc826e215ea4a73ee34de4bbac500cf4fe56eee4f12bdc
+  osm-z9: "openstreetmap-openmaptiles.{{ maps_data_date }}.zoom_0-09.pmtiles"
```

Comment:
I think I'm trending toward these dot-separated formats:

* `<original-name>.<date>.<extension>`
* `<original-name>.<date>.<quality>.<extension>`
* `<original-name>.<date>.extract.<extract-name>.<extension>`

The extract should always be full quality

Reply:
The "bad assumption" was that extract is always full quality. Maybe some day we would have an extract that only includes vector at full quality and satellite at medium quality, or something. But if we hold that assumption, then `<quality>` and `.extract` wouldn't coexist.

Why do you prefer date later? I intuitively feel like it's more fundamental than quality.

Reply:
In that case I would probably would do `full-extract` as the quality to keep the field arrangement more consistent.